docs/source/gravitysensors/tilt.rst

```diff
@@ -1,7 +1,76 @@
 .. include:: ../global.rst
 
-Tilt Sensor Setup
-====================
+Tilt Setup
+==========
 
-.. todo:: Write the Tilt Sensor Setup page
+To associate your a Tilt with Fermentrack, simply place the Tilt near the Raspberry Pi you are using at an angle (not vertical or horizontal) and it will automatically detect it (The Tilt enters a sleep mode when horizontal or vertical).
 
+#. From the main Fermentrack dashboard, click on "Add new gravity sensor"
+#. Click on the Tilt icon
+#. Name the Tilt hydrometer you are adding, select the temperature units you wish to use, and as select the color for easy identification. You also want to specify if you are using a direct Bluetooth connection to your Pi, or using a TiltBridge bluetooth to WiFi adapter.
+
+You will return to the Dashboard and Fermentrack will notify you that a new tilt sensor was added.
+
+Tilt Settings
+-------------
+
+After adding the Tilt to Fermentrack, you are returned to the Gravity Sensors dashboard. Right now, the Tilt is running in unassociated mode in Fermentrack, meaning it is handled as its own device- not integrated with the BrewPi controller. Clicking on the device you just added will bring you into a screen where you can further configure and customize that Tilt sensor. From this page, you can `Manage Sensor`_, `Load Prior Log`_, and `Assign Device`_.
```

Comment:
One of the core tenets of the way that gravity support is built into Fermentrack is that it is intended as a primary feature of the software - on an equal level to support for BrewPi temp controllers.